Source/Diagnostics/ComputeDiagFunctors/BackTransformFunctor.cpp

```diff
@@ -112,14 +114,33 @@ BackTransformFunctor::operator ()(amrex::MultiFab& mf_dst, int /*dcomp*/, const
             const Box& tbx = mfi.tilebox();
             amrex::Array4<amrex::Real> src_arr = tmp[mfi].array();
             amrex::Array4<amrex::Real> dst_arr = mf_dst[mfi].array();
+#ifdef WARPX_DIM_RZ
+            const int nrz = WarpX::n_rz_azimuthal_modes;
+            const int n_rz_comp = 2*nrz - 1;
```

Comment:
Similarly to the comment above:
```suggestion
            const int n_rz_comp = WarpX::ncomps;
```